### DISCUSSION TITLE
hello folk new to nocodb trying to set [Oct 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4068] [Discord Threads]
> 🙋‍♂️ **khurram**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Can you try a simple script with `knex` and your credentials, something as follows:   
####    
#### ```js   
#### const knex = require('knex')({   
####   client: 'mssql',   
####   connection: {   
####     host: 'localhost',   
####     port: 1432,   
####     user: 'sa',   
####     password: 'password',   
####     database: 'test'   
####   }   
#### });   
####    
#### knex.raw('select 1+1').then(res => console.log(res)).catch(e => console.log(e))   
#### ```  
#  
> 🙋‍♂️ **khurram**  
#### ok  
#  
> 🙋‍♂️ **khurram**  
#### has given sysadmin role to sql user; and its working now  
#  
> 🙋‍♂️ **khurram**  
#### earlier i created the needed database (that i specified in NC_DB) and sql user had dbo role to that database only  
#  
> 🙋‍♂️ **khurram**  
#### thanks for the help man! 👍  
#  
> 👨‍💻 **pranavxc**  
#### Glad it helped 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1024925025024876584)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
